src/filePush/filePush.ts

```diff
@@ -34,9 +36,12 @@ export const filePush = async (filepath: string, eventType: string) => {
   const config = await createConfig(eventType);
   logger.info('Created config from secrets');
   const sftp = new Client();
-  const sftpPath =
-    eventType === 'evl' ? process.env.EVL_SFTP_PATH : process.env.TFL_SFTP_PATH;
-  const remoteFileLocation = (sftpPath ?? '') + path.basename(filepath);
+  const sftpPath: { [key in Feeds]: string } = {
+    evl: process.env.EVL_SFTP_PATH,
+    tfl: process.env.TFL_SFTP_PATH,
+  };
+  const remoteFileLocation =
+    ((sftpPath[eventType] as string) ?? '') + path.basename(filepath);
```

Comment:
The casting of `sftpPath[eventType]` to string should mean that this null coalesce never reaches the right hand side. Why was it needed?

Reply:
You may need to update the sftpPath object to return `string | undefined` which is ok as that's what we want when the env var is set to undefined

Reply:
Changed this around, kept this, the idea is if it's undefined then it will be turned to a string so you can still add them safely